bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -407,6 +412,23 @@ static bool nameIdIsNotTransient(Claim c) => c.Type == ClaimTypes.NameIdentifier
         return (user, provider, providerUserId, claims, ssoConfigData);
     }
 
+    /// <summary>
+    /// Provision an SSO-linked Bitwarden user.
+    /// This handles three different scenarios:
+    /// 1. Creating an SsoUser link for an existing User and OrganizationUser
+    ///     - User is a member of the organization, but hasn't authenticated with the org's SSO provider before.
+    /// 2. Creating a new User and a new OrganizationUser, then establishing an SsoUser link
+    ///     - User is joining the organization through JIT provisioning, without a pending invitation
+    /// 3. Creating a new User for an existing OrganizationUser (created by invitation), then establishing an SsoUser link
+    ///     - User is signing in with a pending invitation.
+    /// </summary>
+    /// <param name="provider">The external identity provider.</param>
+    /// <param name="providerUserId">The external identity provider's user identifier.</param>
+    /// <param name="claims">The claims from the external IdP.</param>
+    /// <param name="userIdentifier">The user identifier used for manual SSO linking.</param>
+    /// <param name="config">The SSO configuration for the organization.</param>
+    /// <returns></returns>
+    /// <exception cref="Exception"></exception>
```

Comment:
❓ : should these 2 lines have values?

Reply:
Yep!  Thank you.  My eyes went right past those lines.  I've updated them.